# Contains information about a single gamma ray interaction. Mimics the
# interactionType dtype

float32 energy
float32 x
float32 y
float32 z
float32 dT
float32 dE
uint8 detector
